docker/old-ethermint-docker-compose.yml

```diff
@@ -8,7 +8,6 @@ services:
       - HOME_DIR=/ethermint
     command: [
             "ethermintd", "start",
-            "--home", "/ethermint",
```

Comment:
Is this change related?

Reply:
Yes because if we include the volume it is mounted at `/ethermint` whereas the default location it looks for a config is `/.ethermintd`. But this PR can be ignored in favor of https://github.com/celestiaorg/ephemeral-cluster/pull/22 which also includes this small change.